.github/workflows/docker.yml

```
@@ -30,15 +39,29 @@ jobs:
       - name: Set outputs
         id: vars
         run: |
-          echo "::set-output name=date::$(date +%Y-%m-%d)"
-          echo "::set-output name=sha_short::$(git rev-parse --short HEAD)"
+          echo "sha_short=$(git rev-parse --short HEAD)" >> $GITHUB_OUTPUT
+
       - name: Build and push release version
-        uses: docker/build-push-action@v3
+        if: ${{ github.ref == 'refs/heads/release'}}
+        uses: docker/build-push-action@v4
+        with:
+          file: ./Dockerfile
+          push: true
+          tags: |
+            ghcr.io/marigold-dev/multisig:stable
+            ghcr.io/marigold-dev/multisig:${{ steps.tag_version.outputs.new_version}}-${{ steps.vars.outputs.sha_short }}-release
```

```suggestion
            ghcr.io/marigold-dev/multisig:${{ steps.tag_version.outputs.new_version}}-${{ steps.vars.outputs.sha_short }}-release-prod
```
Is it ok to add that?